manager_config.yaml

```diff
@@ -0,0 +1,66 @@
+manager:
+  private_ip:
+  public_ip:
+  hostname: <manager host name, e.g. manager2>
+  security:
+    ssl_enabled: true
+    admin_password: admin
+
+rabbitmq:
+  ca_path:
+  cluster_members:
+    <first rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+    <second rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+    <third rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+
+postgresql_server:
+  ssl_enabled: true
+  ca_path:
+
+  postgres_password: 'somesecretvalue123!'
+  cluster:
+    nodes:
+      <first postgresql instance-name>:
+        ip:
+      <second postgresql instance-name>:
+        ip:
+      <third postgresql instance-name>:
+        ip:
+
+postgresql_client:
+  ssl_enabled: true
+  server_password: 'somesecretvalue123!'
```

Comment:
This should be randomly generated.

Reply:
Correct, this is taken care of in the next PR.